src/Microsoft.AspNetCore.Dispatcher/SimpleEndpoint.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Dispatcher
+{
+    public class SimpleEndpoint : Endpoint
```

Comment:
Currently unused by the sample, but this is plumbing for a `Use(...)`/`Map(...)` like experience.